concent_api/core/tests/test_unit_transfer_operations.py

```diff
@@ -0,0 +1,44 @@
+from django.test                    import TestCase
+
+from golem_messages                 import message
+from core.transfer_operations       import sign_message
+from utils.testing_helpers          import generate_ecc_key_pair
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = generate_ecc_key_pair()
+
+
+class AddSignatureToMessage(TestCase):
+
+    def setUp(self):
+        self.ping_message = message.Ping()
+
+    def test_add_signature_with_correct_keys_pair(self):
+
+        self.assertEqual(self.ping_message.sig, None)
+
+        ping_message = sign_message(self.ping_message, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
+
+        self.assertIsNot(ping_message.sig, None)
+        self.assertIsInstance(ping_message.sig, bytes)
+
+    def test_empty_arguments_to_add_signature(self):
+
+        with self.assertRaises(AssertionError):
```

Comment:
You're not supposed to test `assert`s. They're not part of functions behavior. It's like trying to get a value of a mathematical function outside of its domain. An assertion on function's input (i.e. a [precondition](https://en.wikipedia.org/wiki/Precondition)) means that the function does not implement any specific behavior the input that fails the assertion and you, as a caller, are responsible for ensuring that it never gets input that triggers it. If the function is meant to handle this case you should not have put that assertion inside it. You should have implemented some well defined behavior for that input instead.